lib/workers/repository/update/branch/auto-replace.spec.ts

```diff
@@ -1,5 +1,6 @@
+import { codeBlock } from 'common-tags';
```

Comment:
> > Does merging this PR mean that you can now get replacements for old deprecated dependencies for all package managers? Of course the replacements still need to be programmed into the code for each package.
> 
> @HonkingGoose merging this PR will add replacement support for all managers _except_:
> 
>     * bazel
>     * git-submodules
>     * gomod
>     * gradle
>     * hermit
>     * homebrew
>     * maven
>     * regex

Are these exceptions listed in the docs somewhere? I don't see any Markdown file changes in this PR. And I also don't see any docs tools code changes.

We should list those exceptions somewhere in our docs.

Reply:
The documentation currently says
```
This config option only works with the npm manager. We're working to support more managers, subscribe to issue [renovatebot/renovate#14149](https://github.com/renovatebot/renovate/issues/14149) to follow our progress.
```

How about changing this to 
```
This config option is currently not working with all managers. We're working to support more managers, subscribe to issue renovatebot/renovate#14149 to follow our progress.
```
and use #14149 to show whats currently working?


Or would you prefer a list directly inside the docs?

Reply:
I prefer a list inside the docs. Can you also put a comment in the code so we know to update the list later?

Reply:
Im not quite sure if thats what you meant but i added a comment at the beginning of the doAutoReplace function.